### PR TITLE
chore: modified login molecule using mui

### LINKED ISSUE #10: Login Page

### PR DESCRIPTION
**Pull request by prtkjakhar**
### TODO
- [x] Bring form vertically center
- [x] add registration page link
- [x] reduce width of form
- [x] Language dropdown
- [x] Sample translation english file
- [ ] Registration molecule #7 
- [ ] Email/password Login molecule #8
- [x] Spinner + disable on login button (after 2 second -> success login)
- [x] Aadhar and mobile validation by digits

Mobile:
Screenshot 2024-02-20 at 5 33 53 PM

### CODE REVIEW

**Comment by prtkjakhar** [comment]
Desktop:
Screenshot 2024-02-20 at 10 36 28 PM

File: molecules/LoginPage/index.tsx [hunk 1]
```diff
@@ -1,46 +1,94 @@
...
-  handleOTPPage,
-}) => {
-  const [input, setInput] = useState("");
-  const handleNumber: React.ChangeEventHandler = (
-    e: React.ChangeEvent<HTMLInputElement>
-  ) => {
-    setInput(e.target.value);
+const LoginPage: React.FC<LoginPageProps> = ({ handleSubmit }) => {
+  const [isAadharClicked, setIsAadharClicked] = useState(false);
+
```
Comment (geeky-abhishek) [inline molecules/LoginPage/index.tsx]: plz use useCallback

File: molecules/LoginPage/index.tsx [hunk 2]
```diff
@@ -1,46 +1,94 @@
...
+            onSubmit={handleSubmit}
+            noValidate
+            sx={{ mt: 1, width: '90%' }}>
+            <TextField
+              margin="normal"
+              required
+              fullWidth
+              label={
+                isAadharClicked ? 'Enter Aadhar Number' : 'Enter Phone Number'
+              }
```
Comment (geeky-abhishek) [inline molecules/LoginPage/index.tsx]: an onChange should be passed here

File: molecules/LoginPage/index.tsx [hunk 3]
```diff
@@ -1,46 +1,94 @@
...
 import styles from './index.module.css';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
 import Image from 'next/image';
-import loginImg from './login.png';
+import logo from './logo.png'
 
 interface LoginPageProps {
```
Comment (geeky-abhishek) [inline molecules/LoginPage/index.tsx]: move it to index.d.ts

**Comment by prtkjakhar** [comment]
### TODO
- [x] Bring form vertically center
- [x] add registration page link
- [x] reduce width of form
- [x] Language dropdown
- [x] Sample translation english file
- [ ] Registration molecule #7 
- [ ] Email/password Login molecule #8
- [x] Spinner + disable on login button (after 2 second -> success login)
- [x] Aadhar and mobile validation by digits

**Comment by prtkjakhar** [comment]
### Preview:
https://github.com/SamagraX-Stencil/ui-templates/assets/65057795/6a23816b-7aa3-4827-8856-f954fd7867e6

**Comment by prtkjakhar** [comment]
Fixes #10

File: molecules/LoginMobileAadharPage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+        <div className={styles.rightColumn}>
+          <div className={styles.topSection}>
+            <div className={styles.dropdown}>
+              <Select
+                size="small"
+                id="lang-select"
+                value={language}
+                label=""
+                onChange={handleLanguage}>
+                <MenuItem value="en">English</MenuItem>
```
Comment (geeky-abhishek) [inline molecules/LoginMobileAadharPage/index.tsx:114]: can we read the language values from config ?

**Pull request closed without merging**